web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -399,6 +400,17 @@ export default defineComponent({
       navigator.clipboard.writeText(value === 'dandiID' ? version : `https://doi.org/${meta.value?.doi}`);
     }
 
+    watchEffect(async () => {
+      // Inject datacite metadata into the page
+      if (meta.value?.doi) {
```

Comment:
@mvandenburgh this seems to currently typecheck due to a catchall clause in the [definition of `DandisetMetadata`](https://github.com/dandi/dandi-archive/blob/c42f6201fabf7cf28a84b1eb52a259d080f90fab/web/src/types/schema.ts#L469). Would it be a good idea to make the `doi` field explicit there?

Reply:
That type is automatically generated from the schema, see https://github.com/dandi/dandi-archive/blob/c42f6201fabf7cf28a84b1eb52a259d080f90fab/web/src/types/schema.ts#L3

Reply:
Oh yeah, thanks for the reminder.